### PR TITLE
Adjustments on CMakeLists.txt to properly install some modules 

### PR DESCRIPTION
**Pull request by bjmarfito**
- Adjustments on CMakeLists.txt to properly install ISSI and filter modules
- Make some adjustments on Stack.py

### CODE REVIEW

File: components/isceobj/Filter/CMakeLists.txt [hunk 1]
```diff
@@ -1,4 +1,16 @@
+Python_add_library(filter MODULE
+	bindings/filtermodule.cpp
+	src/cpxPhaseFilter.cpp
+	src/EdgeFilter.cpp
+	src/Filter.cpp
+	src/filterPhase.cpp
+	src/GaussianFilter.cpp
+	src/MeanFilter.cpp
+	src/medianFilter.cpp
+        )
```
Comment (rtburns-jpl) [inline components/isceobj/Filter/CMakeLists.txt:10]: Please use 4 spaces of indentation here

**Comment by bjmarfito** [comment]
Pls. check @rtburns-jpl if I addressed your suggestions. I've made some changes on the CMakeLists.txt regarding the installation of ISSI.py to the applications folder.

**Comment by bjmarfito** [comment]
@rtburns-jpl, just added it now. Apologies, I've overlooked it.

**Comment by bjmarfito** [comment]
Yes, @rtburns-jpl. Your suggestions worked.

**Comment by rtburns-jpl** [comment]
Great! I think you can also remove contrib/issi/applications/CMakeLists.txt and contrib/issi/components/CMakeLists.txt as they are no longer used.

**Comment by bjmarfito** [comment]
I've deleted those CMakeLists.txt files @rtburns-jpl. Thank you for your suggestions.

**Review by rtburns-jpl — APPROVED** [review]
Looks good, thank you!

**Pull request merged**